com.boxy_svg.BoxySVG.yaml

```diff
@@ -0,0 +1,108 @@
+app-id: com.boxy_svg.BoxySVG
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "21.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: boxy-svg
+tags: [proprietary]
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=host
```

Comment:
this is a very broad filesystem permission

Reply:
I copied it from [Inkscape](https://github.com/flathub/org.inkscape.Inkscape/blob/master/org.inkscape.Inkscape.json) which is also an SVG editor. Restricting the access only to few folders in HOME should work for 99% of users, but I would prefer to no deal with the complains from the 1%.

Reply:
no. this is way too broad. Period.

Reply:
Is it OK if I restrict the filesystem access to xdg-desktop, xdg-documents, xdg-download, xdg-pictures, xdg-templates and xdg-data?

Reply:
yes

Reply:
Done.